### DISCUSSION TITLE
Feedback for Translation of Spanish Discussions

### DISCUSSION THREAD

**Discussion by mjacobus** [community/community#6534] [Discussions]
Hello! 👋
This is a discussion for gathering user feedback on the beta release of Spanish translations for Discussions.

To learn more about this feature and how to use it, see the [changelog post](https://github.blog/changelog/2021-07-19-machine-translation-for-discussions-content/).

Please use this discussion to share your experience using the new feature, and any ideas you have on how the experience could be improved.

**Comment by Newmat20** [comment]
Funciona bien la traducción al español 👍

**Reply by betaadev** [reply]
Functiona bastante bien 👍
It works really good 👍

**Reply by DevWolf31** [reply]
Funciona bien, es muy útil  👍👌!
It works fine, is very good 👍👌!

**Reply by SAMbb25** [reply]
Excelente muy bueno

**Reply by kris0278** [reply]
si, bien

**Reply by discapacidad5** [reply]
Exelente partes mi yo no hablo inglés siempre tenía que ir a Google ahora faltaría la opción para traducir la respuesta que uno hace en español al inglés para que también las personas puedan leer en inglés lo que uno escribe en español

**Reply by myccid** [reply]
el concepto y la traducion ... exelente, la respuesta al problema ... muy mala ...

**Reply by F-D-X** [reply]
Samsun s20

**Reply by F-D-X** [reply]
Mucho  hack

**Reply by ArvizuAutoCenter** [reply]
Dont now how works but hope some one can help thanks

**Reply by Clainetbif** [reply]
Stojn rap find del proccs terminal

**Reply by Josealbet** [reply]
Es bueno solo que guardan toda la información de una manera estricta se supone que es libre y la beta como que obliga ASER cosas que la vdd no se requieren pero es buena por qué tienes 3rramientas nuevas antes de salir más creo que si es como poseedora

**Reply by benetusss** [reply]
benetusss@gmail.com

**Comment by Yllelder** [comment]
This functionality is fantastic. Thank you very much.

**Comment by CreativeUsernameAttempt** [comment]
Aunque entiendo ingles, esta funcion sirve muy bien y puede ayudar a gente que es nueva con el ingles. Gracias por esto!

**Comment by Lite6G** [comment]
Ta piola

**Reply by iakzs** [reply]
ta epico

**Reply by hinojosad429** [reply]
![Uploading Screenshot_20231231-195230.png…]()

**Comment by ghost** [comment]
Gracias a dios, ahora no tendre que abrir otra ventana en el traductor

**Reply by RotrixLOL** [reply]
lo mismo digo xd

**Comment by omnidotnet** [comment]
Muy buena, pero creo que el traductor no entiende contexto. Me tradujo "saving" a ahorro, cuango quería decir "guardar" o "guardado"

**Comment by Lucioric2000** [comment]
El traductor debería aceptar mejoras del texto traducido por parte de hispanohablantes. Inclusive esas ediciones se podrían usar para entrenar las computadoras mediante machine learning para que dén mejoras traducciones en un futuro.

**Comment by BobbyESP** [comment]
Works very very well!

**Comment by emmaviolet** [comment]
:wave: Looking for feedback on translated READMEs

I’ve been looking at supporting READMEs in multiple languages within a single repo, and I’m curious about a) whether this would help you, and b) how you’d most like to interact with such a feature. Bearing in mind that none of these thoughts are at all final, I’d love to know:
- If a README in your language was available in repos you use, would that be helpful? Would you contribute translations for READMEs or other documentation in projects you use often?
- How would you assume you’d toggle between english language docs and translated docs, if you wanted to? Where would you look?
- If a translated README was slightly out of date versus an english README, would that stop you using it? How “out of date” feels like “too” out of date?
- Where translations are available, would you prefer READMEs to be translated by default, with the option to opt-out; or would you prefer English by default, with the translation highlighted?

**Reply by Cachond1** [reply]
I know that it was part of this why I own something... But I don't know what to do, I don't know anything. I need explanation of everything . English is not my native language and I think I have committed inappropriate actions. If there is a minute for me in your time... please, I would appreciate it if you could tell me where I can go.

**Comment by humber42** [comment]
Work fine

**Comment by sebitamontana** [comment]
es muy muy muy muy muy muy muy muy muy muy buena para solo beta

**Comment by mamg22** [comment]
Just tried it, seems very nice to have. I don't have much use for it since I'm fine with english as is, though It will come in handy when I deal with non-english text.

Can there be a option to hide it in the three dots menu? I don't quite like having the translate button on every comment.

**Reply by emmaviolet** [reply]
Looking into this!

**Reply by emmaviolet** [reply]
@mamg22 - just checking in here. It sounds like if you hide the translate button, you'd still want to make it available in the three dots menu so you could access it when needed? Out of interest, do you think you'd _ever_ need to use this, or might it be better to just remove the translation option everywhere in your case?

**Reply by mamg22** [reply]
Hi, thanks for your response. Regarding the first question, that is what I was thinking about. And now that you mention it, being able to remove it sounds like a viable option, as I don't think I'll have a need for this functionality. Anyway, I think either of those two is fine.

**Comment by gingerchicken** [comment]
Seems to work pretty good but is there any way to automatically translate comments and stuff (as in without having to click the button each time)?

**Reply by emmaviolet** [reply]
Yes! We should do this. Watch this space :-)

**Comment by CesarDavidMorenR** [comment]
> Hello! 👋 This is a discussion for gathering user feedback on the beta release of Spanish translations for Discussions.
> 
> To learn more about this feature and how to use it, see the [changelog post](https://github.blog/changelog/2021-07-19-machine-translation-for-discussions-content/).
> 
> Please use this discussion to share your experience using the new feature, and any ideas you have on how the experience could be improved.

Me encanto! Útil para quien no sepa ingles

**Comment by autosquash** [comment]
I think it is great the fact that the translated text doen't replace the original comment. So we have both in view, and obvious translate errors can easily be tracked in the original in order to find out the actual meaning.

**Comment by rafnixg** [comment]
It is a very good translation

**Comment by wolfarmoon** [comment]
Nice! 
I suggest a feature to _manually translate_ md files and wiki pages to avoid auto-translation mistakes.

**Reply by emmaviolet** [reply]
Thanks! This is a great idea and one I've been thinking about. Would love to get your input on the plans if you're up for chatting @tDracoYT - if you email me at emmaviolet@github.com we can find a time.

**Reply by Lucioric2000** [reply]
I suggest a feature to edit the automated translations

**Reply by emmaviolet** [reply]
@Lucioric2000 thanks! Nice idea :-)

**Comment by Wither362** [comment]
- Me encanta! Está súper bien hecho, si hay algún fallo os lo comentaré por aquí abajo, ¿ok?
- I'm lovin 'it! It is super well done, if there is any mistake I will comment it below, ok?

**Comment by rimopa** [comment]
Honestly I, as a Spanish, think this is not in accordance with my goals, this should show the word that means the word in the other language so that users can learn English easily, but most of the pages have all the text in English so the users would need to make more effort to understand the text. People need English.

**Reply by Vanessaaui** [reply]
Está disponible en todos los  idiomas es lo que refleja en la plataforma talvez deverian actualizar

**Comment by yracnet** [comment]
Parese buena opcion, pero veo q esta comenzando a traducir bloques de codigo y estructuras json.

**Comment by 90PabloRomero** [comment]
Looking good!

**Comment by guidospadavecchia** [comment]
Muy buena!

**Comment by Ezekiel1349** [comment]
Excelente, muchas gracias por esto.

**Comment by RipperGOT** [comment]
Wow!!!!!!!! Nice!!!!!  Please apply also in Pull Request.

**Comment by valolastra** [comment]
Hello, I want to unlock my mib std2 pq and I couldn't find qnx neutrino 6.5, I was able to extract the emmc image, I converted the .img to .vmdk, but I don't have qnx, someone can help me, I could also send the image to could someone add toolbox, thanks.

**Comment by Lucioric2000** [comment]
???

Enviado desde mi Samsung Mobile de Telcel
Obtener Outlook para Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by one-new-start** [comment]
Esta muy buena la traducción, también deberían de añadir una función de auto traducción sin tener que darle a traducir

**Comment by hyunsu4020** [comment]
works very good!

**Comment by autosquash** [comment]
¿Cómo puede activarse esta opción en otros repositorios/ámbitos? Me refiero a, por ejemplo, traducir el contenido de un issue.

(Editado para clarificar la pregunta)

**Comment by josueisaihs** [comment]
Works good!

**Comment by discapacidad5** [comment]
Excelente falta ahora la opción de traducir al inglés lo que uno está escribiendo para que se publique en inglés o también a el botón de traducir al inglés dentro del mensaje que esté escrito en español

**Comment by Trollhunters501** [comment]
Funciona bien en español 😎

**Reply by Trollhunters501** [reply]
? Cómo comprar?

**Reply by Trollhunters501** [reply]
En realidad creo que tú texto no le entiendo muy bien como riesgo de vacío?

**Reply by Trollhunters501** [reply]
En realidad el traductor me funciona bien talvez tienes un dispositivo antiguo o un navegador que no tiene cosas actuales

**Comment by Dadiselef20** [comment]
Muchas gracias por hacer esto posible muy buenaobcion

**Comment by Vanessaaui** [comment]
Hola? A todos estamos trabajando en las solicitudes y prevenir fraude prometo que todos los usuarios tendrán su aprobación pronto buenas noches bendiciones ,🙏🙏

**Comment by MillhioreBT** [comment]
Excellent functionality, I really like it
Maybe they can improve the traditions a little more and also add this functionality to all areas of github

**Comment by EliasAfara** [comment]
#ad

Check out [**Contribution Streak Maintainer**](https://www.npmjs.com/package/contribution-streak-maintainer), your go-to solution for effortlessly maintaining your GitHub contribution streak. 🚀

[![npm](https://img.shields.io/npm/v/contribution-streak-maintainer)](https://www.npmjs.com/package/contribution-streak-maintainer) [![npm](https://img.shields.io/npm/dm/contribution-streak-maintainer)](https://www.npmjs.com/package/contribution-streak-maintainer) [![GitHub License](https://img.shields.io/github/license/devful/contribution-streak-maintainer)](https://github.com/devful/contribution-streak-maintainer/blob/main/LICENSE) [![GitHub Repo stars](https://img.shields.io/github/stars/devful/contribution-streak-maintainer)](https://github.com/devful/contribution-streak-maintainer/stargazers)

Do you struggle to keep up with your GitHub contribution streak? Is making daily manual contributions challenging? Say goodbye to those worries!

**Contribution Streak Maintainer** is a powerful npm package designed for seamless integration into your GitHub Actions workflow. By automating the generation and pushing of commits to your repository, this tool ensures your GitHub activity remains active, even on days when you're unable to make manual contributions.

### Key Features

- 📅 Automatic GitHub contribution streak maintenance
- 🛠️ Easy integration into GitHub Actions workflows
- 🌐 Supports a variety of workflows with customizable conditions
- 🤖 Effortlessly keeps your streak alive, allowing you to focus on your work

**Comment by rogeruiz** [comment]
Que chimba está nueva actualización. Está muy útil pa’ cuando quiero entender mejor vainas técnicas. 🥳

**Comment by alphanumericaracters** [comment]
Esta buena la actualización, pero le falta 'contexto', es mu literal la traducción.

The Favaloro Script [bypass master]  = El guión de Favaloro [maestro de derivación] 

- No lo veo a Favaloro escribiendo un guión sobre derivadas (siquiera en la peor alucinación de un LLM). 🤣🤣🤣

Hay muchas palabras que por más que tengan 'traducción' no se deben traducir, como así también que hay palabras que según el contexto tienen uno u otro significado, sin mencionar las que solo tienen significado 'cultural' [region o país].

**Comment by Crisantomartinez369** [comment]
Como la utilizo y como descargarla

**Comment by PushoDev** [comment]
Funciona todo muy bien, es muy útil y agiliza mas las cosas, entiendo bien el ingles pero hay ocasiones que es muy extensa la documentación ya en el idioma nativo fluye mejor la comunicación

**Comment by arteze** [comment]
Tradujo bug como bicho, sé que los bug son bichos, pero acá un bug son problemas.

**Reply by Jonydeere1** [reply]
Exacto!

**Comment by Lucioric2000** [comment]
Aquí el bot no entendió que master es un nombre de rama de Git

**Reply by arteze** [reply]
Podría traducirse como Maestra, o Rama Maestra, o Camino Maestro.

**Comment by casana9002** [comment]
Saludos, este error es al momento de confirmar el pago.
StripeException(error: LocalizedErrorMessage(code: FailureCode.Failed, localizedMessage: , message: null, stripeErrorCode: null, declineCode: null, type: null))

**Comment by KCHARRO** [comment]
Great..... Excellent function.....

**Comment by mlibre2** [comment]
> [!IMPORTANT]
> de maravilla 10 de 10

**Comment by luquefv** [comment]
Veo que el foro tiene temas de discusión en español muy antiguos y poca ayuda. Yo soy nuevo y veo que el programa es mas dificil de lo que pensaba... Hay alguien en mi misma situacion u otros foros o paginas que ayuden a la comunidad en español?

**Reply by Cachond1** [reply]
Yo si... vas a tener que aprender sólito... y conmigo si puedo ayudarte... aquí generalmente no se ayuda... hasta las gracias y por favor son en código..... 
y tienes razón... son muy pocos los repo o archivos o documentos qué están en Español y que tienen coherencia... si puedes ayudar con algunas traducciones , hazlo , hay que ser solidarios empaticos.
respetuoso ... aquí debes de ser muy respetuoso... aunque algunos entes disfrazan sus groserías y en veces son hasta racistas... (no todos .. algunos si).... 
animo.. tu puedes 
y pregunta... pregunta mucho.. siempre hay alguien que escuche....

**Reply by juaniascencion43-hub** [reply]
_ok_

**Reply by juaniascencion43-hub** [reply]
Okey sulo

**Comment by Yahir-ux** [comment]
No me deja exportar mi structura en un mcpack

**Comment by Izai187** [comment]
Que tal escribo para solicitar información sobre cómo puedo contactar con el soporte técnico para implementar acciones en mis repositorios personales y solucionar problemas en mi cuenta usuario isai187. Mi objetivo es evitar riesgos a gran escala y me gustaría contar con su ayuda para asegurar que mis repositorios estén configurados de manera óptima y segura.

Agradezco de antemano su atención y pronta respuesta.

Saludos cordiales,
Moisés

**Comment by Izai187** [comment]
Les escribo para solicitar información sobre cómo puedo contactar con el soporte técnico para implementar acciones en mis repositorios personales y solucionar problemas. Mi objetivo es evitar riesgos a gran escala y me gustaría contar con su ayuda para asegurar que mis repositorios estén configurados de manera óptima y segura.

Además, he experimentado problemas técnicos y bloqueos al intentar acceder o solucionar problemas relacionados con asuntos legales y propiedades personales, incluyendo marcas, organizaciones, licencias y problemas de cuenta. Recientemente, he detectado posibles irregularidades y agradecería su asistencia para investigar y resolver estas cuestiones.

**Reply by Cachond1** [reply]
Me pasa algo similar, espero que se solucionen tus problemas

**Comment by Cachond1** [comment]
Si . La traducción fue geniaaaal)... sobre todo cuando quieres hacer algo y tienes que ver toooodos los comentarios..... de verdad fue genial... sin sarcasmo.. solo que al tener que dar nuestra opinión se pierde algo de lo que estamos haciendo... me entiendes? Me explico?

**Comment by germangar** [comment]
Funciona muy bien, pero una cosa muy obvia que arreglar es que traduce "fix" como "fijar" y en el contexto de github la traducción primaria debería ser clarísimamente "arreglar"
